flink-filesystems/flink-s3-fs-hadoop/pom.xml

```diff
@@ -33,6 +33,7 @@ under the License.
 	<packaging>jar</packaging>
 
 	<properties>
+		<!-- Do not change this without updating the copied Configuration class! -->
```

Comment:
should we add a similar comment to the presto hadoop dependency?

Reply:
done (also the other suggestions)